spmd/tensor/ops/pointwise_ops.py

```diff
@@ -120,10 +118,7 @@
 #     "xlogy",
 # ]
 
-pointwise_ops = [
-    "aten.relu.default",
-    "aten.gelu.default",
-]
+pointwise_ops = ["aten.relu.default", "aten.gelu.default"]
```

Comment:
are we using the same formatter command? `black --line-length 80 spmd test/spmd`? it seems I use this command on my end but it never reformatted like this?

Reply:
Yes I use "black --line-length 80 spmd test/spmd" and it just reformat like this...